internal/runtime/client/client.go

```diff
@@ -364,8 +366,9 @@ func (c *client) CallExtension(ctx context.Context, hook runtimecatalog.Hook, fo
 
 	// If the received response is a failure then return an error.
 	if response.GetStatus() == runtimehooksv1.ResponseStatusFailure {
-		log.Error(err, "extension handler returned a failure response")
-		return errors.Errorf("failed to call extension handler %q: got failure response with message %q", name, response.GetMessage())
+		log.Error(err, "extension handler returned a failure response with message: %v", response.GetMessage())
```

Comment:
```suggestion
		log.Error(err, fmt.Sprintf("failed to call extension handler %q: got failure response with message %v", name, response.GetMessage()))
```
to be consitent with the error message and the other log above
I think it's good if the error message contains the name of the handler
previous version would have panic'ed IIRC because only a key of a k/v pair was set

Reply:
As we use log.Error here. Should we also use it in l.145?

Reply:
I think we should use log.Info in both - here the error isn't actually picking anything up - I think it's always nil.